src/classes/PluginManager.js

```diff
@@ -634,9 +634,15 @@ class PluginManager {
 
     if (!this.hasPlugin(pluginId)) return true;
 
-    if (safe && (!this.canPluginBeDisabled(pluginId)
-        || !this.disablePlugin(pluginId, false).includes(pluginId))) {
-      return false;
+    if (safe && !this.isPluginEnabled(pluginId)) {
```

Comment:
This is a boolean error I think, checking whether the plugin can be disabled after this makes no sense